modules/apps/foundation/portal-security/portal-security-antisamy/src/main/resources/META-INF/resources/sanitizer-configuration.xml

```diff
@@ -490,6 +490,7 @@ http://www.w3.org/TR/html401/struct/global.html
 
 	<allowed-empty-tags>
 		<literal-list>
+			<literal value="img" />
```

Comment:
This change is strange. Previously we didn't have to whitelist `img` tag.

This can cause regressions.

When by defining `<allowed-empty-tags>` we must now list all possibly empty tags then this fix is incomplete and will possibly break other tags like `<br />`

@ambrinchaudhary @csierra Have you tested `<br/>`? What about some other solution like https://lists.owasp.org/pipermail/owasp-antisamy/2010-July/000326.html ? 

Thanks.